CoreFoundation/Base.subproj/CFSortFunctions.c

```diff
@@ -52,8 +52,6 @@ enum {
 
 #define __checkint_int64_mul(x, y, err)         (x * y)
 #define __checkint_uint64_add(x, y, err)        (x + y)
-#define __checkint_int32_mul(x,y,err)           (x * y)
```

Comment:
Hi, while this is not required on 64bits platforms it should still be on arm32 and x86 (committed by me with #1396).

Reply:
I'll fix it!